src/pybind/mgr/rook/module.py

```diff
@@ -126,7 +126,7 @@ def available(self):
             return False, "ceph-mgr not running in Rook cluster"
 
         try:
-            self.k8s.list_namespaced_pod(self._rook_env.cluster_name)
+            self.k8s.list_namespaced_pod(self._rook_env.namespace)
```

Comment:
What does `list_namespaced_pod()` do? I don't see its definition.

Reply:
It returns list of pods for a given namespace. Here we are using it to check if we can connect to the rook pods via kubernetes api.  It is defined here https://raw.githubusercontent.com/kubernetes-client/python/master/kubernetes/client/api/core_v1_api.py

Reply:
I see, we're just checking if we can connect to the k8s api and discarding the results. I'd suggest a simpler check such as if the namespace exists, but not critical.

Reply:
We are checking if namespace exists here:
https://github.com/ceph/ceph/blob/a6c6c114afa9cdc89508ae90bbe837421914d342/src/pybind/mgr/rook/module.py#L125-L126

And it's defined here:
https://github.com/ceph/ceph/blob/a6c6c114afa9cdc89508ae90bbe837421914d342/src/pybind/mgr/rook/module.py#L87-L88